DOCS.md

```diff
@@ -447,16 +460,38 @@ Nested key example:<br />
       headline = 'one-time'
     }
   }
+  -- or
+  {
+    e = {
+      description = 'Event',
+      subtemplates = {
```

Comment:
This is not something that exists in Emacs (AFAIK), but considering it's an optional improvement I'm ok with leaving it.